database/store.py

```diff
@@ -3,7 +3,7 @@
 from operator import itemgetter
 from typing import List
 
-from config import NOTIFY_TEMPLATE_SUBMIT_APPLICATION
+from config.default import Default as Config
```

Comment:
Importing from default means that the specific environment config won't be used. Perhaps use current_app.config? Alternatively there is a patter in [Authenticator](https://github.com/communitiesuk/funding-service-design-authenticator/tree/main/config) whereby environment-specific config is available via `from config.env import env` even if current_app is not available (i.e. outside the request context.